lib/redis.js

```diff
@@ -256,13 +256,17 @@ Redis.prototype.connect = function (callback) {
         reject(err);
         return;
       }
+      var CONNECT_EVENT = 'connect';
+      if (_this.options.tls) {
+          CONNECT_EVENT = 'secureConnect';
+      }
```

Comment:
It would be better to use `var CONNECT_EVENT = _this.options.tls ? 'secureConnect' : 'connect';` instead